client/src/js/base/__tests__/ProgressBar.test.js

```diff
@@ -1,95 +1,48 @@
-import { ProgressBar, AutoProgressBar } from "../ProgressBar";
+import { ProgressBar, AffixedProgressBar } from "../ProgressBar";
 
-describe("<AutoProgressBar />", () => {
-    let props;
-    let state;
-
-    beforeEach(() => {
-        props = {
-            step: 1,
-            bsStyle: "danger",
-            active: true,
-            interval: 2,
-            affixed: true
-        };
-        state = {
-            fill: 0,
-            active: true
-        };
-    });
+describe("<ProgressBar />", () => {
+    let props = {
+        now: 0,
+        bsStyle: ""
+    };
 
-    it("should render when [this.state.fill=0]", () => {
-        const wrapper = shallow(<AutoProgressBar {...props} />);
-        wrapper.setState({ fill: 0 });
+    it("should render", () => {
+        const wrapper = shallow(<ProgressBar {...props} />);
         expect(wrapper).toMatchSnapshot();
     });
 
-    it("should render when [this.state.fill!=0]", () => {
-        const wrapper = shallow(<AutoProgressBar {...props} />);
-        wrapper.setState({ ...state, fill: 1 });
+    it("should render when [bsStyle='success']", () => {
+        const wrapper = shallow(<ProgressBar {...props} />);
         expect(wrapper).toMatchSnapshot();
     });
 
-    it("componentDidUpdate() should update this.interval when [this.state.fill !== prevState.fill] and [this.state.fill === 10]", () => {
-        const wrapper = shallow(<AutoProgressBar {...props} />);
-        wrapper.setState({ ...state, fill: 10 });
-        wrapper.setProps({ ...props, interval: 10000 });
-        expect(wrapper.instance().interval).toEqual(9);
-    });
-
-    it("componentDidUpdate() should call window.clearInterval when [this.props.active=false && prevState.active=true]", () => {
-        const wrapper = shallow(<AutoProgressBar {...props} />);
-        wrapper.instance().interval = 1;
-        window.clearInterval = jest.fn();
-        wrapper.setProps({ props, active: false });
-        expect(window.clearInterval).toHaveBeenCalledWith(1);
-    });
-
-    it("componentWillUnmount() should call window.clearInterval when component unmount", () => {
-        const wrapper = mount(<AutoProgressBar {...props} />);
-        wrapper.instance().interval = 1;
-        window.clearInterval = jest.fn();
-        wrapper.unmount();
-        expect(window.clearInterval).toHaveBeenCalledWith(1);
+    it("should render when [bsStyle='danger']", () => {
+        const wrapper = shallow(<ProgressBar {...props} />);
+        expect(wrapper).toMatchSnapshot();
     });
 });
 
-describe("<ProgressBar />", () => {
-    let props;
+describe("<AffixedProgressBar />", () => {
+    let props = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['props' is never reassigned. Use 'const' instead.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5093312)